change/react-native-windows-dba9665a-a1e1-4b98-824c-21504ac6d547.json

```diff
@@ -0,0 +1,7 @@
+{
+  "type": "patch",
+  "comment": "Lib project will use the app's ExperimentalFeatures.props",
```

Comment:
Consider something user-facing for the changelog, like "Fix community module template solution"